tests/baselines/reference/jsFileMethodOverloads3.errors.txt

```diff
@@ -0,0 +1,30 @@
+/a.js(2,5): error TS7010: 'overload', which lacks return-type annotation, implicitly has an 'any' return type.
+/a.js(7,5): error TS7010: 'overload', which lacks return-type annotation, implicitly has an 'any' return type.
+
+
+==== /a.js (2 errors) ====
+    /**
+     * @overload

```

Comment:
@DanielRosenwasser Is it acceptable to attach an error to the tag name? Or is it better to include it in the whole tag? Can we use "overload" in the error message?

Reply:
I would do the entire tag, and I'd create a new diagnostic like

```
This overload implicitly returns the type 'any' because it lacks a return type annotation.
```

Reply:
Thanks